packages/server/api/controllers/users.controller.js

```diff
@@ -33,6 +33,11 @@ const createUser = async (body) => {
   }
 };
 
+const getUserInformation = (uID) => {
+  return knex('users').select('id').where({ uid: uID });
```

Comment:
oh, I know why you have two different ids now :)

Usually it is better to not expose database uuid's if there is another ID to do with. Or just stick to one id per db entry.